src/OrbitGl/App.cpp

```diff
@@ -313,6 +313,11 @@ void OrbitApp::OnCaptureFinished(const CaptureFinished& capture_finished) {
         ORBIT_UNREACHABLE();
         break;
     }
+
+    if (capture_data_ != nullptr && capture_data_->file_path().has_value()) {
+      capture_file_info_manager_.AddOrTouchCaptureFile(capture_data_->file_path().value(),
+                                                       GetCaptureTime());
+    }
```

Comment:
When I am reading this I have the following questions. 
1. Should that maybe only get executed when the capture was successful? Maybe also in the case `kInterruptedByService`? 
2. Why this `capture_data_ != nullptr` check? Can capture_data_ be a nullptr here, or not? Should this be a ORBIT_CHECK? Maybe and ORBIT_DCHECK? Maybe just an ORBIT_ERROR in an else branch?

Reply:
I think this should be consistent with when we create the capture file output stream. 

We created the capture file when capture started and we stream data to the capture file. This happens before processing the capture finished event. 

If we execute this only for the cases of successful and `kInterruptedByService`, then we are doing a quite different thing as before when we add and touch file while capture started.